### PR TITLE
feat(cf-component-select): migrate select to fela

### PR DESCRIPTION
**Pull request by koddsson**
Migrate the `cf-component-select` component to use the CSS-in-JS
framework fela.

Closes: #213

### CODE REVIEW

File: packages/cf-component-select/src/Select.js [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+  width: theme.width,
+  display: theme.display,
+  verticalAlign: theme.verticalAlign
+});
+
+const Label = createComponent(
+  () => ({
+    display: 'block',
+    fontSize: '0.7em',
+    color: '$cf-select-label-color',
```
Comment (tajo) [inline packages/cf-component-select/src/Select.js]: ¯\_(ツ)_/¯

Comment (koddsson) [inline packages/cf-component-select/src/Select.js]: lol

File: packages/cf-component-select/test/Select.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
-import Select from '../../cf-component-select/src/index';
+import { Select } from '../../cf-component-select/src/index';
 
 test('should render', () => {
   const tree = shallow(<Select />);
```
Comment (tajo) [inline packages/cf-component-select/test/Select.js]: `mount()` would be better. 

1. the snapshots contains only the Fela wrapper
2. `shallow()` doesn't pass context (theme) down correctly

Comment (koddsson) [inline packages/cf-component-select/test/Select.js]: Sure

**Pull request merged**